source/localizable/tutorial/subroutes.md

```diff
@@ -0,0 +1,425 @@
+An index nested route works similarly to the base index route.
+It is the default route that renders when no route is provided.
+Therefore in our case, when we navigate to `/rentals`, Ember will attempt to load the rentals index route as a nested route.
+
+To create an index nested route, run the following command:
+
+```shell
+ember g route rentals/index
+```
+
+If you open up your Router (`app/router`) you may notice that nothing has updated.
+
+```app/router.js
+Router.map(function() {
+  this.route('about');
+  this.route('contact');
+  this.route('rentals');
+});
+```
+
+Much like how our application's `index` route doesn't appear in our Router, `index` routes on sub-routes won't explicitly appear in the Router either.
+Ember knows that the default action is to take the user to the `index` route.
+However, you can add the `index` route if you want to customize it.
+For example, you can modify the `index` route's path by specifying `this.route('index', { path: '/custom-path'})`.
+
+In the section on [using Ember Data](../ember-data#toc_updating-the-model-hook), we added a call to fetch all rentals.
+Let's implement our newly generated `rentals/index` route by moving this `findAll` call from the parent `rentals` route to our new sub-route.
+
+```app/routes/rentals.hbs{-2,-3,-4}
+export default Ember.Route.extend({
+  model() {
+    return this.store.findAll('rental');
+  }
+});
+```
+
+```app/routes/rentals/index.js{+2,+3,+4}
+export default Ember.Route.extend({
+  model() {
+    return this.store.findAll('rental');
+  }
+});
+```
+
+Now that we are returning all of our rentals to the nested route's model, we will also move the rental list markup from our main route template to our nested route index template.
+
+```app/templates/rentals.hbs{-9,-10,-11,-12,-13,-14,-15,-16,-17}
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <h2>Welcome!</h2>
+  <p>We hope you find exactly what you're looking for in a place to stay.</p>
+  {{#link-to 'about' class="button"}}
+    About Us
+  {{/link-to}}
+</div>
+{{#list-filter
+   filter=(action 'filterByCity')
+   as |rentals|}}
+  <ul class="results">
+    {{#each rentals as |rentalUnit|}}
+      <li>{{rental-listing rental=rentalUnit}}</li>
+    {{/each}}
+  </ul>
+{{/list-filter}}
+{{outlet}}
+```
+
+```app/templates/rentals/index.hbs{+1,+2,+3,+4,+5,+6,+7,+8,+9}
+{{#list-filter
+   filter=(action 'filterByCity')
+   as |rentals|}}
+  <ul class="results">
+    {{#each rentals as |rentalUnit|}}
+      <li>{{rental-listing rental=rentalUnit}}</li>
+    {{/each}}
+  </ul>
+{{/list-filter}}
+{{outlet}}
+```
+
+Finally, we need to make our controller that has our filter action available to the new nested index route.
+Instead of moving the whole controller file over to `app/controller/rentals/index.js` from `app/controller/rentals.js`, we'll just take advantage of the route's `controllerName` property to just point at our existing `rentals` controller.
```

Comment:
So if I didn't do this I would have to add a good sized section explaining that we need to remove this controller, create a new controller, and move the logic.  I think its easier to do it this way, plus you are introducing a potentially useful feature.  I don't think its really that awkward since the controllers attached to the parent route. which makes sense.